public/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>TimeStamp</title>
+    <link rel="stylesheet" href="/css/main.css" />
+  </head>
+  <body>
+    <div class="center"><h1>TimeStamp Microservice</h1></div>
+
+</br>
+</br>
+<div class="center">
+<h1>Example Usage:</h1>
```

Comment:
This should be indented

Reply:
I updated the html format hopefully it is fixed